views/partials/head.ejs

```diff
@@ -1,16 +1,16 @@
-<!doctype html>
-<html lang="nl">
-	
-<head>
-	<meta charset="UTF-8">
-	<meta name="author" content="CMD Online">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-		
-	<link href="<% baseURL %>/public/styles/style.css" rel="stylesheet">
-
-	<link rel="preconnect" href="https://fonts.googleapis.com" />
-	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-	<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;600;700&display=swap" rel="stylesheet"/>
-	
-	<title>Team Creator</title>
+<!doctype html>
+<html lang="nl">
+	
+<head>
+	<meta charset="UTF-8">
+	<meta name="author" content="CMD Online">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+	<link href="../../public/styles/style.css" rel="stylesheet">

```

Comment:
As discussed last night, this is causing issues for some reason, but like you said I think that once I convert my team generator stuff into routes it should (hopefully) solve itself hehe